pkg/kiosk/kiosk.go

```diff
@@ -60,6 +71,7 @@ func Kiosk(cfg *Config) {
 	if err != nil {
 		panic(err)
 	}
+
```

Comment:
one last nitpick: Here are too many empty lines spread around the code for my taste :D